features/steps/thens/alignment.py

```diff
@@ -93,7 +110,7 @@ def step_impl(context, inst, ifc_rep_criteria, ifc_layout_criteria):
                         if (item.is_a().upper() == "IFCCOMPOSITECURVE") and (
```

Comment:
```python
case ("IfcCompositeCurve as Axis", "absence of IfcAlignmentVertical and IfcAlignmentCant"):
``` 
In line 106 (I think it's still part of ALA001). 

`observed = "', '".join(entities.split(" and "))
`

This will return the string 
'expected=None'
'observed={'OneOf' : ['IfcAlignmentVertical', 'IfcAlignmentCant']}

Which I think would be clear